collector/mscluster_network.go

```diff
@@ -0,0 +1,116 @@
+package collector
+
+import (
+	"github.com/StackExchange/wmi"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	registerCollector("mscluster_network", newMSCluster_NetworkCollector) // TODO: Add any perflib dependencies here
```

Comment:
Remove the comment here, perflib isn't in use.

Reply:
Done